nameable.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'person'
+
+# class Nameable
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+# class Decoration < Nameable
+class Decorator < Nameable
+  def initialize(nameable)
+    super
```

Comment:
- [ ] Kindly change this to `super()` for proper inheritance.

Reply:
@julie-ify I have already added super here in the Decorator so I didn't understand your comment for this specific one. Can you please explain this particular comment?